### PR TITLE
chore(core): remove unnecessary updateLastSignInAt

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Remove unnecessary updateLastSignInAt.

## Testing

UTs.

### CODE REVIEW

File: packages/core/src/routes/session/username-password.ts [hunk 1]
```diff
@@ -94,8 +94,8 @@ export default function usernamePasswordRoutes<T extends AnonymousRouter>(
         passwordEncrypted,
         passwordEncryptionMethod,
         roleNames,
+        lastSignInAt: Date.now(),
       });
-      await updateLastSignInAt(id);
```
Comment (gao-sun) [inline packages/core/src/routes/session/username-password.ts:98]: is this function still in use?

Comment (darcyYe) [reply packages/core/src/routes/session/username-password.ts:98]: Yes. In some sign-in cases, we do not update other users information in DB but only the `lastSignInAt` field. Or we can simply delete this method and use `updateUserById`. I'm good with both.

Comment (charIeszhao) [reply packages/core/src/routes/session/username-password.ts:98]: Only use `updateUserById` seems good enough to me

**Review by charIeszhao — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

File: packages/core/src/routes/session/username-password.ts (hunk 1 above)
Comment (darcyYe) [reply packages/core/src/routes/session/username-password.ts:98]: use `updateUserById ` instead of all `updateLastSignInAt`.

**Pull request merged**